app/messages/page.tsx

```
@@ -82,6 +85,106 @@ function MessagesPage() {
+              if (update.displayName && !conv.participantDisplayName) {
+                updated = updated === conv ? { ...conv } : updated
+                updated.participantDisplayName = update.displayName
+                changed = true
+              }
+              return updated
+            })
+            return changed ? next : prev
+          })
+        }
+      } finally {
+        idsToHydrate.forEach(id => participantHydrationInFlightRef.current.delete(id))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)